composer.json

```
@@ -12,8 +12,10 @@
     }
   },
   "scripts": {
-    "cbf": "vendor/bin/phpcbf --standard=PSR2 src",
-    "lint": "vendor/bin/phpcs --standard=PSR2 src",
+    "cbf": "vendor/bin/phpcbf -s --standard=phpcs.ruleset.xml --extensions=php ./src/",
```

Thoughts on changing this to `lint:phpcbf` to match the command [elsewhere](https://github.com/pantheon-systems/smart_content_cdn/blob/16e7bef8be49d81d0b6cd71aa7badec2f8cc2b0b/composer.json#L28)? Not a blocker IMO.

---

Not married to the way it is now. It made more sense before I added more `lint:` commands 😅

```suggestion
    "lint:phpcbf": "vendor/bin/phpcbf -s --standard=phpcs.ruleset.xml --extensions=php ./src/",
```